generate.sh

```diff
@@ -29,6 +29,10 @@ pip install -q -r requirements.txt
 
 # Setting up documentation sources
 sphinx-quickstart --ext-githubpages -q -v $VERSION -a $AUTHOR -p $PROJECTNAME -t templates/ -d html_theme=${DOCTHEME:-alabaster} -d html_logo=${LOGO:-} > /dev/null
+if [ $? -ne 0 ]; then
+  echo -e "Failed to setup source directory.\n"
```

Comment:
Always open for suggestion. 
I just interpreted it from the description of `sphinx-quickstart`